### ISSUE TITLE
Suggest to start Test Management Tool

### ISSUE THREAD

**Issue by ViktoriaNeva** [levelupik/levelupik#23]
I think would be great to start writing test cases for LL before it become difficult and will have not much test cases to create. 
1. In the next releases we could do smoke tests to check if everything works
2. Plus with increasing functionality we will add test cases with no missing. 
3. Now it will be simpler to start creating test cases with simple functionality

I suggest TestLink 
It’s free and open source
https://www.softwaretestinghelp.com/testlink-tutorial-1/

**Comment by levelupik** [comment]
@ViktoriaNeva great idea.
Let's make a discussion.
Would you like, please, to describe a little bit of this QA stuff?

This TestLink system should help QA persons to have a place where they can store a ToDo plan?
Is it request much functionality to make our own system?
Or TestLink has some big and hard in implementation killer feature, with what it is making a QA life much easier?

Is it possible to make some sketches of your vision this perfect test case place?

**Comment by ViktoriaNeva** [comment]
@levelupik 
Yes. Sure. I need some time and I’ll provide you sketches and document. 

@VetGam
Do you have any ideas about that? Please, your opinion in this question will be very helpful. What do you think QAs will need from Test Management System?

**Comment by ViktoriaNeva** [comment]
@levelupik I was thinking through this question and for project like this (with no big team and having free contribution in the testing), LL does not need Test Designs or Test Plans, we need only possibility to write test cases and do test suits.

1. Tester can Create Test Case
    - Title
    - Steps
    - Expected Result
    - Priority
    - In the Test Suit? If yes - In which?
2. Create Test Suits (like Smoke, Functionality or by category Registration, Notes etc)
    - Combine Suits with existing test cases
    - Assign to (if LL has new release lets assign to someone to so smoke)
    - Start new test (there tester starting testing and can mark each test case by pass or failed)
    - It would be cool to make functionality for viewing previous testing (failed, passed)
4. In the future smoke tests should be automated. So LL could start tests whenever it needs without the testers outside. 

Do you think this will be possible to write this functionality soon?
